src/map/entities/battleentity.h

```diff
@@ -143,7 +143,8 @@ enum SKILLTYPE
 	SKILL_ALC			= 55,
 	SKILL_COK			= 56,
 	SKILL_SYN			= 57,
-	SKILL_RID			= 58
+	SKILL_RID			= 58,
+	SKILL_DIG			= 59 // Chocobo digging

```

Comment:
don't think either of them are in polutils (they have no name, so why would they be in the client?), but i don't think it really matters.. i would have used riding because I don't think there is actually a skill for riding anyways